forms-shared/tests/summary-email/renderSummaryEmail.ts

```diff
@@ -2,6 +2,7 @@ import { generatePageScreenshot } from '../../test-utils/generatePageScreenshot'
 import { toMatchImageSnapshot } from 'jest-image-snapshot'
 import { getExampleFormPairs } from '../../src/example-forms/getExampleFormPairs'
 import { renderSummaryEmail } from '../../src/summary-email/renderSummaryEmail'
+import { mapValues } from 'lodash'
```

Comment:
I try to keep lodash imports like this
`import mapValues from 'lodash/mapValues'

but we need eslint for this I think.